src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +17,28 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> line = new ArrayList<String>();
+        try {
+            File file = new File(fileName);
+            FileReader fileReader = new FileReader(file);
+            BufferedReader reader = new BufferedReader(fileReader);
+            String readedLine;
+            while ((readedLine = reader.readLine()) != null) {
+                String[] convertedLine = readedLine.toLowerCase().replaceAll("[^a-zA-Z,\" \"]+", "")
+                        .split(" ");
+                for (String word : convertedLine) {
+                    if (word.startsWith("w")) {
+                        line.add(word);
+                    }
+                }
+            }
+            if (line.isEmpty()) {
```

Comment:
can we create a solution without this check?

Reply:
ok